ttnn/cpp/ttnn/operations/conv/conv2d/conv2d.cpp

```diff
@@ -668,6 +668,9 @@ Result conv2d_L1(
             activation,
             compute_config);
 
+        if (conv_config.deallocate_activation) {
```

Comment:
what happens if input was RM, and got tilized at line 635?
You should extend the regression test to cover several variants 
input tensor Tiled/RM input tensor sharded/interleaved